cloud/openstack/os_networks_facts.py

```diff
@@ -33,10 +33,16 @@
     - "python >= 2.6"
     - "shade"
 options:
-   network:
+   name:
      description:
-        - Name or ID of the Network
+        - Name of the Network
      required: false
+     version_added: "2.2"
+   id:
```

Comment:
This parameter is not not used by the module, and is not needed.

The name parameter can accept either name or id.